spec/functional/resource/user/useradd_spec.rb

```diff
@@ -77,9 +76,19 @@ def password_should_be_set
     end
   end
 
+  def try_cleanup
+    ['/home/foo', '/home/bar'].each do |f|
+      shell_out("rm", "-r", f)
+    end
+    ['cf-test'].each do |u|
+      shell_out("userdel", "-r", username)
```

Comment:
if this is the case, the after block is incorrect https://github.com/chef/chef/blob/jdm/useradd-specs/spec/functional/resource/user/useradd_spec.rb#L99